src/poker/game/GamePlay.java

```diff
@@ -9,26 +10,25 @@ public class GamePlay {
     public HandsAndBoard result;
     public Player player1;
     public Player player2;
-    GameState gameState;
-    List<Player> playerList;
-    public EquityEvaluator equityEvaluator;
+    public GameState gameState;
+    public List<Player> playerList;
+    public static EquityEvaluator equityEvaluator;
```

Comment:
taka klasa nigdy nie pwoinna byc statikiem, taka klasa jest za bardzo dynamiczna (nie chce mi sie nawet myslec zeby to dorze uargumetnowac czegos takiego nigdy sie nie pisze w kodzie.)